packages/hash/frontend/src/pages/[shortname]/types/entity-type/[entity-type-id].page/definition-tab/property-list-card.tsx

```diff
@@ -40,15 +57,282 @@ import {
   EntityTypeTableCenteredCell,
   EntityTypeTableHeaderRow,
   EntityTypeTableRow,
-  EntityTypeTableTitleCellText,
 } from "./shared/entity-type-table";
 import { InsertTypeRow, InsertTypeRowProps } from "./shared/insert-type-row";
-import { MultipleValuesCell } from "./shared/multiple-values-cell";
+import {
+  MULTIPLE_VALUES_CELL_WIDTH,
+  MultipleValuesCell,
+} from "./shared/multiple-values-cell";
 import { QuestionIcon } from "./shared/question-icon";
 import { TypeFormModal } from "./shared/type-form";
-import { TYPE_MENU_CELL_WIDTH, TypeMenuCell } from "./shared/type-menu-cell";
+import { TypeMenuCell } from "./shared/type-menu-cell";
 import { useStateCallback } from "./shared/use-state-callback";
 
+const CollapsibleTableRow = ({
+  expanded,
+  depth,
+  lineHeight,
+  children,
+}: {
+  expanded: boolean;
+  depth: number;
+  lineHeight: number;
+  children: ReactNode;
+}) => {
+  return (
+    <TableRow>
+      <TableCell colSpan={12} sx={{ p: "0 !important", position: "relative" }}>
+        <Collapse
+          in={expanded}
+          sx={{
+            position: "relative",
+            top: `-${lineHeight}px`,
+            mb: `-${lineHeight}px`,
+          }}
+          appear
+        >
+          <Box
+            sx={{
+              position: "absolute",
+              height: lineHeight,
+              width: "1px",
+              left: `${13.4 + 20 * depth}px`,
```

Comment:
Is there anything we can do to make this less magical? What is it related to?

Reply:
Got rid of it in f5073edca4b96d5972f46218d39365e4c9afe26d

Reply:
This is still magic to me – it appears in two components with identical values, but they aren't connected. 

Is it a problem if they become out of sync? If so, we should have some function to generate it rather than duplicate it.

Also, can `10 * 20` be derived from some other value we have access to? Or is it just to match the design?

This isn't a big deal but if there's an easy way to ensure we don't update something and this breaks, let's do it – whether 'something' is one of these components being updated without the other one, or some other value somewhere that affects this

Reply:
Thanks for the explanation, that makes sense. I think I fixed it in 69a24d5ccd38aa6d925d38e0cd5a44158d2504ea.